pyproject.toml

```diff
@@ -19,13 +19,12 @@ classifiers = [
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
 ]
-requires-python = ">=3.9"
+requires-python = ">=3.10"
 dependencies = [
     "aiofiles",
     "emoji",
     "fastapi",
     "httpx",
-    "importlib_metadata>=4.6; python_version<'3.10'",
```

Comment:
No longer needed, since we at least have 3.10 and thus don't need a backport of `importlib.metadata`.